CMakeLists.txt

```diff
@@ -88,7 +88,10 @@ if(CMAKE_BUILD_TYPE MATCHES "Release")
         -sENVIRONMENT=worker
     )
 else()
-    set(CLP_FFI_JS_EXTRA_LINKER_FLAGS -sENVIRONMENT=node)
+    set(CLP_FFI_JS_EXTRA_LINKER_FLAGS
+        -sENVIRONMENT=node
+        -sNO_DISABLE_EXCEPTION_CATCHING
```

Comment:
I haven't looked into it myself, but would [`-fwasm-exceptions`](https://emscripten.org/docs/porting/exceptions.html?highlight=no_disable_exception_catching#emscripten-javascript-based-exception-support) also work?

Reply:
Both [`-fexceptions`](https://emscripten.org/docs/porting/exceptions.html?highlight=no_disable_exception_catching#emscripten-javascript-based-exception-support) and [`-fwasm-exceptions`](https://emscripten.org/docs/porting/exceptions.html?highlight=no_disable_exception_catching#webassembly-exception-handling-based-support) work. 

Enabling exception catching with `-fwasm-exceptions` seems to introduce minor performance impacts. i.e. Decoding all log events in a sample log file with ~380k log events 5 times takes 10.486s without the flag and 10.504s with the flag.

On the other hand, with JS-emulated exceptions enabled by `-fexceptions`, decoding time increases significantly to 17.087s.

As a result, we should enable `-fwasm-exceptions`.